src/java.base/share/classes/java/lang/Math.java

```diff
@@ -3314,60 +3319,25 @@ public static float nextDown(float f) {
      * @since 1.6
      */
     public static double scalb(double d, int scaleFactor) {
-        /*
```

Comment:
FWIW, I wrote this code a long time ago prior to OpenJDK. IIRC, some of the complications of this method may have been due to wanted to avoid declaring the method `strictfp` back when  `strictfp` vs default-FP semantics were an issue.